public/sites/default/all.services.yml

```diff
@@ -7,3 +7,9 @@ parameters:
     - 'avustukset.dev.hel.ninja'
     - 'avustukset.stage.hel.ninja'
 
+services:
+  resilient_logger.service:
+    class: ResilientLogger\ResilientLogger
+    factory: ['Drupal\helfi_audit_log\ResilientLogger', 'createFromSettings']
+    arguments:
+      - '@settings'
```

Comment:
I think this should be defined in [helfi_audit_log.services.yml](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/blob/dev/public/modules/custom/helfi_audit_log/helfi_audit_log.services.yml), so it is easier to find.